internal/tiltfile/helm_test.go

```diff
@@ -61,3 +61,36 @@ func TestHelmUnknownVersion(t *testing.T) {
 
 	assert.Equal(t, expected, actual)
 }
+
+const fileRequirementsYAML = `dependencies:
+  - name: foobar 
+    version: 1.0.1
+    repository: file://./foobar`
+
+func TestLocalSubchartFileDependencies(t *testing.T) {
```

Comment:
i don't find this to be a super compelling test case. I'd like to see multiple dependencies,  some with `repository` values that match the prefix and some that don't, and when you check the output of `localSubchartDependencies`, assert the length as well to make sure that nothing got in there that shouldn't have.

Reply:
oh huh i see, you tested things that don't match the prefix below.